server/routes/admin.js

```diff
@@ -4,13 +4,9 @@ import AuthorController from '../controllers/AuthorController';
 
 const admin = express.Router();
 const BASE_URL = '/admin';
-const { AuthorValidator } = middlewares;
-// const { verifyToken, authorizeUser, AuthorValidator } = middlewares;
 const { addAuthor } = AuthorController;
-const { authorValidation } = AuthorValidator;
-// const authorizeAdmin = authorizeUser(['admin']);
```

Comment:
Let's uncomment this line and user it on line 14

Reply:
Done